src/search/components/SearchResult/index.js

```diff
@@ -26,16 +41,35 @@ export default class SearchResult extends React.Component {
   render() {
     const { result } = this.props;
     return (
-      <Link to={`/search/addons/${result.slug}`} className="search-result" ref="container">
-        <div className="search-result--name" ref="name">
-          {result.name}
+      <li className="SearchResult" ref="container">
+        <div>
+          <img
+            className="SearchResult-icon"
+            src={result.icon_url}
+            height="64"
```

Comment:
Should be good to leave these out since as they're specified in the CSS that should prevent a reflow anyway.